db/migrate/20220126135321_add_last_activity_at_to_user.rb

```diff
@@ -0,0 +1,14 @@
+class AddLastActivityAtToUser < ActiveRecord::Migration[6.1]
+  def change
+    ActiveRecord::Base.record_timestamps = false
+
+    add_column :users, :last_activity_at, :datetime
+    add_index :users, :last_activity_at
```

Comment:
While it might not make a big difference for this table, you'll usually end up with a faster migration by adding a column index _after_ populating that column with data.

Reply:
@holmesie did you mean to say that in response to my question as an alternative regarding an upgrade from MySQL 5.7 to 8?

Reply:
@timwongj nope, just happened to see this and thought I'd leave a comment. Hope everyone is doing well!

Reply:
gotcha, thanks for the feedback and hope you're doing well too!

Reply:
[d47d38ecc4c96ce0195e297bd478189b0c2829f5](https://github.com/SaraAlert/SaraAlert/pull/1200/commits/d47d38ecc4c96ce0195e297bd478189b0c2829f5)